autoload/lsp/options.vim

```diff
@@ -33,7 +33,9 @@ export var lspOptions: dict<any> = {
   # Show a diagnostic message on a status line
   showDiagOnStatusLine: false,
   # Don't print message when a configured language server is missing.
-  ignoreMissingServer: false
+  ignoreMissingServer: false,
+  # Make diagnostics show in a popup instead of echoing
+  showDiagInPopup: true
```

Comment:
Can you add also update the lsp.txt help file with a description of this new option?

Reply:
Yeah, i'll do that

Reply:
Done, i've added